services/recipeservice/src/main/java/ch/whatshouldieat/services/recipeservice/controller/RecipeController.java

```diff
@@ -0,0 +1,71 @@
+package ch.whatshouldieat.services.recipeservice.controller;
+
+import java.util.List;
+import java.util.Optional;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import ch.whatshouldieat.services.recipeservice.model.Recipe;
+import ch.whatshouldieat.services.recipeservice.service.RecipeService;
+
+@RestController
+public class RecipeController {
+    
+    private final RecipeService service;
+  //  private final String CORS_IP_ALLOW = "http://ac04efe572a774d43ae8144e0b735fc5-2046315918.eu-central-1.elb.amazonaws.com:80";
```

Comment:
Is this comment needed? Eventually a security issue? Could this be outsourced to a properties file ?

Reply:
removed